frontend/src/app/libs/common-components/components/tab-navigation/models/tabs.ts

```diff
@@ -0,0 +1,5 @@
+export enum Tabs {
+  'generalTab',
```

Comment:
set string values to better readability in templates, `Tabs { General: 'general' ... }`